pydotorg/tests/test_context_processors.py

```diff
@@ -24,3 +25,12 @@ def test_url_name(self):
 
         mock_request = MockRequest(path='/nothing/here/')
         self.assertEqual({}, context_processors.url_name(mock_request))
+
+    def test_blog_url(self):
+        """
+        test the blog_url context processor
+        :return:
```

Comment:
It looks like this was accidentally added :) Also, we can remove the whole docstring. The test itself is pretty understandable.